### DISCUSSION TITLE
Feature Preivew - Mangal WEB UI

### DISCUSSION THREAD

**Discussion by metafates** [metafates/mangal#184] [Ideas]
Just a small feature preview on the new Web UI that mangal v5 will have. 👀

https://github.com/metafates/mangal/assets/62389790/93082deb-4886-4aef-9166-2b33a6be5be4

**Comment by luevano** [comment]
Pretty cool. Just noticed you didn't leave mangal to die. Any thoughts on accepting collaboration of any kind in mangalorg (lua providers for now maybe)?

**Reply by metafates** [reply]
Sure! There will be a main repository for the lua providers here https://github.com/mangalorg/saturno

I plan to rewrite [these providers](https://github.com/metafates/mangal-scrapers) and I would accept a help, of course. However, new providers api currently lacks proper documentation

**Comment by OmiChrome** [comment]
i would reallly really love to try it :) 
this soooo much awesome work <3
